docs/source/specs/openapi.json

```diff
@@ -1331,6 +1331,15 @@
             "schema": {
               "type": "string"
             }
+          },
+          {
+            "name": "service-accounts",
+            "in": "query",
+            "description": "A comma separated list of usernames for service accounts to remove from the group",
+            "required": true,
```

Comment:
I think the best thing to do would be to make both query parameters optional (not required).

Reply:
Done!